src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -299,6 +327,7 @@ docker pull {{ jobData.image }} \
 ## network consumption
 docker run --name $docker_name \
   --init \
+  --detach \
```

Comment:
We still think removing `--rm` to achieve oom checker is too costly.

Reply:
Other change is fine.

Reply:
In this PR it is only used to determine what is (or was) the container `cgroup` to scan `dmesg` log for that value. I'm open for better solutions preserving `--rm`. Technically, we can write `cgroup` name to a file from the task container and the read it in exit handler.

Reply:
Agree, I think we need to implement it in this PR as well, given that currently PAI has no service to reap trash.